app/javascript/spec/ansible-edit-catalog-form/mockdata.js

```diff
@@ -0,0 +1,6124 @@
+export const initalDataForFirstTestCase = {
+    "catalogFormId": "148",
+    "allCatalogs": [
+        [
+            "My Company/AWS",
+            19
+        ],
```

Comment:
Feel like this much mock data is unnecessary. I think it's ok to merge as is for now, but I think we should remove a lot of this in a followup pr.